### PR TITLE
[4] add plugin icon in quickicon

### PR DESCRIPTION
**Pull request by PhilETaylor**
### Summary of Changes

Add + action for plugins to go to the installer. 

### Testing Instructions

Home Dashboard

### Actual result BEFORE applying this Pull Request

Screenshot 2021-05-14 at 21 03 03

### Expected result AFTER applying this Pull Request

Screenshot 2021-05-14 at 21 31 26

### Documentation Changes Required

none

### CODE REVIEW

File: administrator/language/en-GB/mod_quickicon.ini [hunk 1]
```diff
@@ -27,6 +27,7 @@ MOD_QUICKICON_MODULE_MANAGER="Modules"
 MOD_QUICKICON_MODULE_MANAGER_ADD="Add Site Module"
 MOD_QUICKICON_NAV_LABEL="Quick Links"
 MOD_QUICKICON_PLUGIN_MANAGER="Plugins"
+MOD_QUICKICON_PLUGIN_MANAGER_ADD="Install new plugin"
```
Comment (Quy) [inline administrator/language/en-GB/mod_quickicon.ini]:
```suggestion
MOD_QUICKICON_PLUGIN_MANAGER_ADD="Install New Plugin"
```

Comment (PhilETaylor) [reply administrator/language/en-GB/mod_quickicon.ini]: Thanks.

**Comment by brianteeman** [comment]
I am not in favour of this PR. The plus on a module creates a new instance of a module it does not install a module.

**Comment by PhilETaylor** [comment]
or maybe this icon

Screenshot 2021-05-14 at 21 28 45

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ff35ade794f785b42cd847b61b145c5bc51e4046<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33883">issues.joomla.org/tracker/joomla-cms/33883</a>.</sub>

**Comment by brianteeman** [comment]
Still not in favour for the same reasons.

**Comment by PhilETaylor** [comment]
Im going to change the icon as I think its better anyway...

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ff35ade794f785b42cd847b61b145c5bc51e4046<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33883">issues.joomla.org/tracker/joomla-cms/33883</a>.</sub>

**Comment by HLeithner** [comment]
Hmm only superuser can install Plugins/extensions. Shouldn't we check the ACL before adding the plus?

I think that's true for all buttons or did I missed the check?

File: administrator/modules/mod_quickicon/src/Helper/QuickIconHelper.php [hunk 2]
```diff
@@ -172,11 +173,13 @@ public function getButtons(Registry $params, CMSApplication $application = null)
...
-					'link'   => Route::_('index.php?option=com_plugins'),
-					'name'   => 'MOD_QUICKICON_PLUGIN_MANAGER',
-					'access' => array('core.manage', 'com_plugins'),
-					'group'  => 'MOD_QUICKICON_SITE'
+					'image'    => 'icon-plug',
+					'link'     => Route::_('index.php?option=com_plugins'),
+					'linkadd'  => Route::_('index.php?option=com_installer'),
+					'linkicon' => 'icon-upload',
+					'name'     => 'MOD_QUICKICON_PLUGIN_MANAGER',
+					'access'   => array('core.manage', 'com_plugins'),
```
Comment (PhilETaylor) [inline administrator/modules/mod_quickicon/src/Helper/QuickIconHelper.php:180]: @HLeithner right here.... for the com_plugins, but the add buttons dont have their own

Comment (PhilETaylor) [reply administrator/modules/mod_quickicon/src/Helper/QuickIconHelper.php:180]: I have added the ACL check explicitly in ac6ff41

**Comment by PhilETaylor** [comment]
I have added the ACL check explicitly in ac6ff41

**Comment by brianteeman** [comment]
I am not against a quicklink to install extensions but to me it makes no sense only to do this for plugins

**Comment by PhilETaylor** [comment]
Well what else would you do it to?

Just send the PRs then 😊

**Comment by brianteeman** [comment]
its introducing a new concept to the quickicon - for what benefit?

**Comment by PhilETaylor** [comment]
The same could be said for most things in Joomla 4. I give up.

**Pull request closed without merging**